src/index.tsx

```diff
@@ -2,5 +2,19 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import createSagaMiddleware from 'redux-saga';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import { reducer } from './redux/reducer';
+
+const saga = createSagaMiddleware();
+
+const store = createStore(reducer, applyMiddleware(saga));
```

Comment:
I would recommend to move `store` and `saga` init configuration into separate folders.
Let me show what I mean.

Create `/src/store` folder and put `store` variable into `root.store.ts` file and `saga` into `root.saga.ts` file.

At the end you should have something like this

```
src/
  store/
    root.store.ts
    root.saga.ts
```